contents/blog/gdpr-google-analytics-privacy-shield.md

```diff
@@ -0,0 +1,82 @@
+
+There's lots of detail in the full ruling (link needed), but here's a quick summary:
+
+- An Austrian website was found to have exported personal data of a user to Google servers in the US
+- The complaint argued this was a breach as it was subject to demands from local intelligence agencies
+- The Austrian Data Protection Agency agreed it was a breach and the case remains ongoing 
+
+That's a very simplified version of events, but here are three important takeaways:
+
+### 1 . Google was cleared of any wrongdoing (for now)
+Only the website that was using Google Analytics, as the data controller, was found liable for the breach. Under the GDPR, breaching the regulation can result in huge fines of up to €20 million or 4% of a company's global turnover, whichever is the largest. [Amazon was fined €746 million by authorities in Luxembourg in 2020](https://www.bbc.co.uk/news/business-58024116), the largest GDPR fine so far. Getting it wrong can be painful.
+
+### 2. Any website using Google Analytics in the EU could be liable
+As confirmed by [guidance](https://tweakers.net/nieuws/192020/autoriteit-persoonsgegevens-waarschuwt-voor-mogelijk-verbod-op-google-analytics.html) published by the Dutch Data Protection Authority, any website or product using Google Analytics in the EU could be in breach of the GDPR. Much of the ruling hinges on specific configuration options within GA, which further complicates things for end users.
+
+### 3. This is just the beginning
+This complaint was but one of 100 similar complaints lodged by the same non-profit group. The founder, Max Schrems, [told WIRED](https://www.wired.co.uk/article/google-analytics-europe-austria-privacy-shield): "It's not specific to Google Analytics. It's basically about outsourcing to US providers in general". Other complaints include the use of Google Analytics and Facebook Connect by the likes of Airbnb and Ikea, so expect to hear about more rulings in the coming months.
+
+## How did we get here? A brief history of GDPR
+Most of the conversation thus far has focused on whether this means [Google Analytics is illegal](https://isgoogleanalyticsillegal.com/) to use in Europe, but this goes way beyond Google Analytics alone. This is just the latest chapter in a long running saga concerning a core principle of the GDPR: data portability.
+
+In its [response](https://blog.google/around-the-globe/google-europe/its-time-for-a-new-eu-us-data-transfer-framework/) to the ruling, Google urged the EU and US to come up with a new legal framework to create legal stability. It's a reasonable demand but, we've been here several times before.
+
+To understand the problem a little better, it helps to understand the history of GDPR. Let's break things down: 
```

Comment:
Any chance you can get this down to 10 - 'a history of GDPR in 10 lessons' or something